File: Dataset/Leetcode/train/70/215.py
class Solution:
    def XXX(self, n):
        """
        :type n: int
        :rtype: int
        """
#       s[n] = s[n-1] + s[n-2]

#       爆了
        # if n <= 2:
        #     return n
        # return self.XXX(n-1) + self.XXX(n-2)

        if n <= 2:
            return n
        
        s = [1, 2]
        for i in range(2,n):
            s.append(s[i-1] + s[i-2])
        return s[n-1]

File: Dataset/Leetcode/train/3/215.cpp
 class Solution {
public:
    int XXX(string s) {
        int maxLength = 0;
        map<char, int> m;
        int len = 0;
        for (int i = 0; i < s.size(); i ++) {
            if (m.count(s[i]) == 0) {
                len += 1;
                m[s[i]] = i;
            } else {
                len = 0;
                i = m[s[i]];
                m.clear();
            }
            if (maxLength < len) {
                    maxLength = len;
                }
        }
        return maxLength;
    }
};

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }

